modules/benchmark/R/calc.benchmark.R

```diff
@@ -0,0 +1,123 @@
+##-------------------------------------------------------------------------------------------------#
+##' For each benchmark id, calculate metrics and update benchmarks_ensemble_scores
+##'  
+##' @name calc.benchmark 
+##' @title Calculate benchmarking statistics
+##' @param settings object from start.benchmark.runs (for now - we many only need a few variables)
+##' @param con database connection
+##' @export 
+##' 
+##' @author Betsy Cowdery 
+
+calc.benchmark <- function(settings, con){ #settings file is output from start.benchmark.runs
+  
+  require(RPostgreSQL)
+  require(XML)
+  require(data.table)
+  
+  
+  # Query BRR for benchmark ids
+  
+  bm.id <- settings.bm$run$id #maybe something like that?
+  
```

Comment:
The function argument is `settings`, so `settings.bm` should throw an error (is this a typo?)